tagstudio/src/cli/ts_cli.py

```diff
@@ -2069,13 +2071,13 @@ def scr_choose_option(self, subtitle: str, choices: list, prompt: str = '', requ
 					# 	# self.cleanup()
 					# 	sys.exit()
 					# Cancel =======================================================
-					if (com[0].lower() == 'cancel' or com[0].lower() == 'c' or com[0] == '0') and required==False:
+					if com_name in ('cancel', 'c', '0') and not required:
 						clear()
 						return -1
 					# Selection ====================================================
-					elif int(com[0]) > 0 and int(com[0]) <= len(choices):
+					elif com_name.isdigit() and 0 < int(com_name) <= len(choices):
```

Comment:
here I added condition `.isdigit()`, otherwise it could end up with `ValueError` when calling `int()` on non-numeric string.